.github/script-missing-mode-enforcement.py

```diff
@@ -0,0 +1,62 @@
+"""
+Script to scan the OpenTTD's script API for functions that miss checks for the
+function being called from the right mode (deity or company mode).
+
+When a function calls either ScriptObject::Command or ScriptObject::GetCompany
+then the function is considered dangerous. When one of the mode enforcement
+macros from script_error.hpp, i.e. EnforceDeityMode, EnforceCompanyModeValid or
+EnforceDeityOrCompanyModeValid, are called in the function, then we consider
+that the function has mode enforcement.
+
+Any dangerous function for which no enforcement is found are emitted as errors.
+"""
+
+import glob
+import re
+import sys
+
+def checkModeEnforcement(path):
```

Comment:
Very nitpicking, feel free to ignore: Python normally uses this_way_of_making_function and similar for this_way_of_variables. Camelcasing really isn't used commonly.